specs/core/0_beacon-chain.md

```diff
@@ -880,7 +880,7 @@ def get_shuffling(seed: Hash32,
     )
 
     # Shuffle
-    seed = xor(randao_mix, Hash32(slot))
+    seed = xor(randao_mix, hash(slot))
```

Comment:
The intent here was a type cast, not a call to the `hash` function.

Reply:
ah